test/config/df-segment.data.xml

```diff
@@ -99,14 +99,51 @@
  <rel name="data_writer" class="DataWriterConf" id="dw-01"/>
  <rel name="uses" class="DFHWConf" id="dfhw-01"/>
 </obj>
+	
+	<obj class="DFApplication" id="df-02">
```

Comment:
Why are these lines indented differently to the other objects?

Reply:
Likely due to a different editor. Let me do a dbe_main pass to clean it up.